CONTRIBUTING.md

```diff
@@ -217,3 +224,145 @@ Please also note that you need to replace `$DOCKER_USER` with your own DockerHub
 Only the maintainers can release a new version of actions-runner-controller, publish a new version of the helm charts, and runner images.
 
 All release workflows have been moved to [actions-runner-controller/releases](https://github.com/actions-runner-controller/releases) since the packages are owned by the former organization.
+
+### Workflow structure
+
+Following the migration of actions-runner-controller to the GitHub actions work, all the workflows had to be modified to accomodate the move to the new organization. The following table describes the workflows, their purpose and dependencies.
```

Comment:
This might be a bit more concise
```suggestion
Following the migration of actions-runner-controller into GitHub actions, all the workflows had to be modified to accommodate the move to the new organization. The following table describes the workflows, their purpose and dependencies.
```